corehq/sql_db/routers.py

```diff
@@ -76,23 +85,29 @@ def allow_migrate(db, app_label, model_name=None):
         return db == settings.SYNCLOGS_SQL_DB_ALIAS
 
     if not settings.USE_PARTITIONED_DATABASE:
-        return app_label != PROXY_APP and db in (DEFAULT_DB_ALIAS, None)
+        return app_label not in (PROXY_APP, PROXY_STANDBY_APP) and db in (DEFAULT_DB_ALIAS, None)
 
     if app_label == PROXY_APP:
-        return db == plproxy_config.proxy_db
+        return (
+            db == plproxy_config.proxy_db
+            or bool(plproxy_standby_config and db == plproxy_standby_config.proxy_db)
```

Comment:
This change will make ALL the SQL functions get created on the proxy 'standby' DB (including the write functions). Calling a 'write' function will error. The alternative is to select only those functions that are reads and have them created on the 'standby' proxy in separate migrations. 

@emord curious if you have an opinion on which is better. I did this initially because it was simpler but now feeling like having all those additional functions around is messy and could lead to confusion.

Reply:
I could see this either way. No matter what there would be some error that bubbles up to sentry.

One thing for the future might be if there were ever a switch to using pglogical, logically replicated database could still be written to. If those write functions were there, then there wouldn't be an error